GNUmakefile

```diff
@@ -1,4 +1,5 @@
-TEST?=$$(go list ./... |grep -v 'vendor')
+TEST?=$$(go list ./azuredevops/internal/acceptancetests |grep -v 'vendor')
```

Comment:
I used `go list ./... | grep acceptancetests` in `accetest.sh` to get all acceptance tests, regardless where the acceptance test directory is located inside the code base